### ISSUE TITLE
flex content inside swipeable panel

### ISSUE THREAD

**Issue by lianeddy** [enesozturk/rn-swipeable-panel#87]
i cant seem to create a view with full height using {flex: 1}. is there a way to do this without using the height property?

**Comment by seyaobey-dev** [comment]
Same here, I'm facing the same issue

**Comment by seyaobey-dev** [comment]
Same here, I'm facing the same issue.

**Comment by isaachinman** [comment]
Anyone ever find a solution here?

**Comment by isaachinman** [comment]
Fairly sure it's due to [this `TouchableHighlight`](https://github.com/enesozturk/rn-swipeable-panel/blob/main/src/Panel.tsx#L269). Is that true, @enesozturk?

**Comment by enesozturk** [comment]
Yes, it's probably because of that. But we have to do like so. If the content will be scrollable (which may be longer than default panel height) we cannot make it full height with `flex: 1`, but if not, you can use `canScroll: false`.

**Comment by rogorman9** [comment]
I know this is old, but that `TouchableHighlight` also breaks the `stickyHeaderIndices` prop in `scrollViewProps`, because the `ScrollView` only ever has a single child when it's scrollable.
